src/com_tjnotifications/admin/views/notification/tmpl/edit.php

```diff
@@ -2,9 +2,17 @@
 
 // No direct access
 defined('_JEXEC') or die;
-JHtml::_('formbehavior.chosen','select');
-JHtml::_('behavior.formvalidator');
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\HTML\HTMLHelper;
+
+HTMLHelper::_('formbehavior.chosen','select');
+HTMLHelper::_('behavior.formvalidator');
+
 $today= gmdate('Y-m-d');
+
+$options['relative'] = true;
+
+HTMLHelper::_('script', 'com_tjnotifications/template.min.js', $options);

```

Comment:
@punambaravkar As you have added template.js move js written on view to template.js